sunpy/map/map_factory.py

```diff
@@ -95,7 +97,16 @@ def _read_file(self, fname, **kwargs):
         # This can be removed once read_file supports pathlib.Path
         log.debug(f'Reading {fname}')
         try:
-            pairs = read_file(os.fspath(fname), **kwargs)
+            if str(fname).endswith("asdf"):
```

Comment:
The code to check the first 5 bytes was in the PR, was it removed?

Reply:
yes that was in my older pr but instead of reading again I chose to stick with existing one that were present shall I add the` first_5bytes =fp.read(5) `  again ?